core/src/main/scripts/importer/validateData.py

```diff
@@ -3764,6 +3809,44 @@ class StudyResourceValidator(ResourceValidator):
     def __init__(self, *args, **kwargs):
         """Initialize a StudyResourceValidator with the given parameters."""
         super(StudyResourceValidator, self).__init__(*args, **kwargs)
+        self.defined_resources = {}
+
+    def checkLine(self, data):
```

Comment:
normally methods use underscores in python, but i see your following the existing convention in this file so that's fine. Just FYI